sql/core/src/main/scala/org/apache/spark/sql/expressions/Window.scala

```diff
@@ -19,7 +19,7 @@ package org.apache.spark.sql.expressions
 
 import org.apache.spark.annotation.Stable
 import org.apache.spark.sql.Column
-import org.apache.spark.sql.catalyst.expressions._
+import org.apache.spark.sql.catalyst.expressions.{WindowSpec => _, _}
```

Comment:
I think this was an error, sort of; the wildcard import shadowed the package's WindowSpec, which seems to be what's intended. The resolution worked incorrectly in 2.13 as a result.

Reply:
Ahh, I see. Will it be nicer to import `WindowSpec`, explicitly?

Reply:
You could; it's implicitly imported from the package. I think it would still collide without this exclusion